waspc/examples/todoApp/ext/queries.js

```diff
@@ -9,7 +9,10 @@ export const getTasks = async (args, context) => {
 
   const Task = context.entities.Task
   const tasks = await Task.findMany(
-    { where: { user: { id: context.user.id } } }
+    {
+      where: { user: { id: context.user.id } },
+      orderBy: { id: 'asc' },
```

Comment:
Ok! Or you could have used creation date? Is id guaranteed to grow with creation time?

Reply:
I wanted to, but it doesn't seem like we are storing it. But the id is set to `autoincrement()`, so we should be good.